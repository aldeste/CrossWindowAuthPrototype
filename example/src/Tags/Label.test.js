import Label from "./Label";
import renderer from "react-test-renderer";
import React from "react";
// BUG: Theres a bug with jest-styled-components and styled-components 2.0.0-17, fix when updated.
// import { matcher, serializer } from "jest-styled-components";
//
// expect.addSnapshotSerializer(serializer);
// expect.extend(matcher);
//
// Once fixed, replace toMatchSnapshot() with toMatchStyledComponentsSnapshot()

describe("Label tag", () => {
  it("Should be defined", () => {
    expect(Label).toBeDefined();
  });

  it("Is a styled component", () => {
    expect(Label.name).toBe("StyledComponent");
  });

  it("Matches earlier configuration", () => {
    const tree = renderer.create(<Label />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
