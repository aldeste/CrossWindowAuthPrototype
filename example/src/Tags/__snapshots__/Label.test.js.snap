// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label tag Matches earlier configuration 1`] = `
<label
  className="sc-bdVaJa QTUjL"
/>
`;

exports[`Label tag Matches earlier configuration if required is passed 1`] = `
<label
  className="sc-bdVaJa gPpDMU"
  required={true}
/>
`;
